tests/stub/bookmarks/test_bookmarks_v4.py

```diff
@@ -1,15 +1,13 @@
-import json
-
 from nutkit.frontend import Driver
 from nutkit.protocol import AuthorizationToken
 from tests.shared import TestkitTestCase
 from tests.stub.shared import StubServer
 
 
-# TODO: Tests for 3.5 (no support for PULL n)
-
 # Tests bookmarks from transaction
-class TestBookmarks(TestkitTestCase):
+class TestBookmarksV4(TestkitTestCase):
```

Comment:
Renaming classes changes the test id an can break pipelines of other drivers if they want to skip a test from this class.  
So far, I've tried to make the transition of drivers that don't have a 4.4 branch yet as smooth as possible by running my changes against their 4.3 branches locally and making sure that they pass.  
If you haven't done that, I'd recommend doing so.

Reply:
I opened a similar PR to 4.3 and it seems like we are ok on this occasion (java is expected to fail at the moment).